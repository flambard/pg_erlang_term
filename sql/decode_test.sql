select erlang_term_decode('\x836a')
-- select erlang_term_decode('\x8383770665726c616e67');
