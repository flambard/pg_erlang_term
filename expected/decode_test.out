select erlang_term_decode('\x836a')
-- select erlang_term_decode('\x8383770665726c616e67');
 erlang_term_decode 
--------------------
 []
(1 row)

